frontend/src/pages/org/collection-detail.ts

```diff
@@ -123,7 +137,7 @@ export class CollectionDetail extends LiteElement {
      @sl-request-close=${() => (this.showPublishedInfo = false)}
   >
       <p class="text-left">
-        Embed this published collection in other site using the following embed code and ReplayWeb.page:
+        Embed this published collection in another website using the following embed code and ReplayWeb.page:
```

Comment:
FYI I refactored this section https://github.com/webrecorder/browsertrix-cloud/pull/974 so this line will create a merge conflict, I've went ahead and fixed the copy in my branch if you want to revert it here!

Reply:
will do, ty!